docs/content/doc/help/faq.en-us.md

```diff
@@ -275,4 +277,16 @@ By default, your LFS token will expire after 20 minutes. If you have a slow conn
 You may want to set this value to `60m` or `120m`.
 
 ## How can I create users before starting Gitea
-Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md" >}}) to add users like normal.
+Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md#admin" >}}) to add users like normal.
+
+## How can I enable password reset
+There is no setting for password resets. It is considered "enabled" when a [mail service]({{< relref "doc/usage/email-setup.md" >}}) is configured, and "disabled" otherwise.
+
+## How can a user's password be changed
+- As an **admin**, you can change any user's password (and optionally force them to change it on next login)...
+  - By navigating to your `Site Administration -> User Accounts` page and editing a user.  
+  - By using the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md#admin" >}}).  
+  Keep in mind most commands will also need a [global flag]({{< relref "doc/usage/command-line.en-us.md#global-options" >}}) to point the CLI at the correct configuration.
+- As a **user** you can change it... 
+  - In your account `Settings -> Account` page (this method **requires** you to know your password).
+  - By using the `Forgot Password` link if there is a configured [mail service]({{< relref "doc/usage/email-setup.md" >}}).
```

Comment:
```suggestion
  - By using the `Forgot Password` link. If the `Forgot Password` link is not available, please contact your administrator to configure a [mail service]({{< relref "doc/usage/email-setup.md" >}}). 
```

Reply:
Manually merged change with small tweak. 

The `Forgot Password` link is always available, but the page it goes to will tell the user if it's disabled or not.  
As well, the page has an `Account Recovery` heading rather than `Forgot Password`